portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,47 @@
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p>Namaste! Welcome to my portfolio. Let's think of human existence 
+          for some seconds. Where are we? How are we here? Have we been able to 
+          understand the reason behind our existence? Let's think through this.
+          In the ever increasing universe, our location is in a tiny 
+          galaxy known as Milky Way. The radius of the Milky Way galaxy is 52,850 light years. 
+          (Remember! light years is a unit of distance not time. 
+          1 light year = 9.461*10^12 km or 5.87*10^12 miles) Now you may wonder
+          why I called it a tiny galaxy. The distance between earth and the 
+          observable edge of the universe is 46 billion light years making the 
+          diameter around 93 Jeff's net worth is 93 billion dollars(Might reach 
+          this number in the future). If we take the fraction of money equivalent
+          to the size of milky way galaxy then it would be around 0.00011365 
+          percent of his net worth.</p>
+      <p>See! How tiny is it? Now, let me co me back to my initial argument. 
+          If milky way is such tiny then you can imagine the 
+          relative size of the solar system in comparison to the universe, the 
+          size of the earth to the solar system, and your size to the earth. 
+          Does it sound fascinating? May be or maybe not? I might have 
+          helped you a little bit in understanding your present existence 
+          and location. Other questions that I raised above are 
+          "Life's Big Questions" and humans since the beginning of evolution 
+          have been trying to answer. In this grand scale, do we really exist?🤔🤔</p>
+         <p> Now, let's set this argument aside (I promise, we will come back 
+          to it again soon.), and jump into the story of a human 
+          being who is living in a lakehouse that abuts the old Tennessee lake. 
+          Currently, this tiny cognitively sensible living being is in front
+           of his laptop trying to understand how to write a Javascript 
+           function as a part of his internship.</p>
+        <p> Wait, I forget to introduce myself. Try any one of them and I will open up to you!</p>
+        <button1 onclick="window.location.href='aboutme.html';">About me</button1>
+        <button2 onclick="window.location.href='blogs.html';"/>My Blog</button2>
+        <button3 onclick="window.location.href='travel.html';"/>Travel and Food</button3>
+        <button4 onclick="window.location.href='books.html';"/>Books</button4>
+        <button5 onclick="window.location.href='projects.html';"/>My Projects</button5>       
```

Comment:
I thought these were styled differently and ended up confusing you (and me). I think we can just go with button tags:

```suggestion
        <button onclick="window.location.href='aboutme.html';">About me</button>
        <button onclick="window.location.href='blogs.html';"/>My Blog</button>
        <button onclick="window.location.href='travel.html';"/>Travel and Food</button>
        <button onclick="window.location.href='books.html';"/>Books</button>
        <button onclick="window.location.href='projects.html';"/>My Projects</button>       
```